test: Only remove route/dns/route_rule before test

cathay4t opened the pull request:
Since nmstate support changing DNS/route/route_rule without interface
defined, there is no need to include full state of all interface to
clean up the test environment.

cathay4t commented:
wait PR #1621

ffmancera reviewed (dismissed):
Thank you! This is a great change before that was required but not anymore.

cathay4t commented:
Investigating the CI failure about `None.get_uuid()` which has stable reproducer now.

cathay4t committed:
test: Only remove route/dns/route_rule before test

Since nmstate support changing DNS/route/route_rule without interface
defined, there is no need to include full state of all interface to
clean up the test environment.

Updated the integration test case because container will have eth1
unmanaged by NetworkManager which will still hold route and DNS config.

Signed-off-by: Gris Ge <fge@redhat.com>

ffmancera reviewed (approved):
Thanks!
